### PR TITLE
Fix readme and TODO

### PR DESCRIPTION
**Pull request by xakep-amatop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,10 +41,10 @@ For simulation `qemu_x86` or `qemu_x86_64` board can be used.
 For xen based system `xenvm-qemu` board can be used.
```
Comment (xakep-amatop) [inline README.md:41]: note: I think that it is needed to update this line too, there isn't board with name `xenvm-qemu`...

**Review by firscity — APPROVED** [review]
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

with one comment commit topics + message for "Sync README with new changes": I think they should be more specific (e.g. "Update README with CONFIG_DOMD_*_PATH options"), but I am not the maintainer here. IMHO, it will simplify search.

File: TODO.md [hunk 2]
```diff
@@ -1 +1,2 @@
 1. Make mAosError in protocol as aos::ErrorInfo in order to provide info about file name and line
+2. Enable DMA for SD/MMC driver after resolving https://github.com/xen-troops/zephyr/issues/68#issuecomment-1914729489 issue.
```
[the remark this answers is not in the document]
Comment (xakep-amatop) [reply TODO.md:2]: ok, I'll drop this change

File: README.md (hunk 1 above)
Comment (xakep-amatop) [reply README.md:41]: what about other platforms mentioned above, e.g. `native_posix_64`, `qemu_x86_64` and so on?

**Comment by xakep-amatop** [comment]
@firscity, please take another look at this PR

Note: I deleted the configuration (cfg) and overlays for all QEMU boards. I need confirmation from your side that this action is not prohibited.

**Review by al1img — APPROVED** [review]

**Review by firscity — APPROVED** [review]

**Review by al1img — APPROVED** [review]

**Pull request merged**